metadata/Cadastre/basic_parcels/washington_county_parcels_schema.md

```diff
@@ -0,0 +1,11 @@
+# Title
+
+Washington County Parcels Field Definitions
+
+## ID
+
+94694d7c-95d3-4bcc-a8ed-265a59ce5077
+
+### ACCOUNT_NUM
```

Comment:
I think we can remove ACCOUNT_NUM now that it has been added to all the counties and has become part of the Basic parcel schema